rocAL/rocAL/source/pipeline/node.cpp

```diff
@@ -23,11 +23,12 @@ THE SOFTWARE.
 #include "node.h"
 Node::~Node()
 {
-    if(!_node)
-        vxReleaseNode(&_node);
+    if(!_src_tensor_roi) vxReleaseTensor(&_src_tensor_roi);
+    if(!_dst_tensor_roi) vxReleaseTensor(&_dst_tensor_roi);
+    if(!_node) vxReleaseNode(&_node);
     _node = nullptr;
-    vxReleaseTensor(&_src_tensor_roi);
-    vxReleaseTensor(&_dst_tensor_roi);
+    _src_tensor_roi = nullptr;
```

Comment:
Can we make 
_scr_tensor_roi = nullptr;
_dst_tensor_roi = nullptr;
_node = nullptr;

In order same as how we release the pointers